content/en/data_jobs/airflow.md

```diff
@@ -170,13 +180,19 @@ For Astronomer customers using Astro, <a href=https://www.astronomer.io/docs/lea
 [7]: https://docs.datadoghq.com/account_management/api-app-keys/#api-keys
 [8]: https://github.com/OpenLineage/OpenLineage/releases/tag/1.23.0
 [9]: https://www.astronomer.io/docs/astro/runtime-provider-reference#astro-runtime-1210
-{{% /tab %}}
-
-{{< /tabs >}}
+[10]: https://www.astronomer.io/docs/astro/environment-variables/#management-options
+[11]: https://www.astronomer.io/docs/astro/manage-env-vars#using-your-dockerfile
 
 ## Validation
 
-In Datadog, view the [Data Jobs Monitoring][2] page to see a list of your Airflow job runs after the setup. 
+In Datadog, view the [Data Jobs Monitoring][2] page to see a list of your Airflow job runs after the setup.
+
+
+## Troubleshooting
+Check that the OpenLineage environment variables are correctly set on the Astronomer deployment. Note that using the `.env` file to add the environment variables does not work because the variables are only applied to the local Airflow environment.
```

Comment:
```suggestion
Check that the OpenLineage environment variables are correctly set on the Astronomer deployment.

**Note**: Using the `.env` file to add the environment variables does not work because the variables are only applied to the local Airflow environment.
```